CHANGES

```diff
@@ -1,7 +1,7 @@
 Version 1.5.3
 -------------
 
--
+- Fix for serializing null values in Event extras map.
```

Comment:
More details about what this affects (buffering? network serialization? … etc.) would be helpful here.

Reply:
Just buffering, but good call.